terraform/variables.tf

```diff
@@ -35,6 +35,11 @@ variable "vertex_region" {
   type        = string
 }
 
+variable "bigquery_location" {
```

Comment:
How does this work with `transfer_datasets.sh`, for EU/non-EU locations? Pls update the docs (I think `README.md`) if they need updating

Reply:
`transfer_datasets.sh` is used to copy data from a US region to EU so no need to update documentation.

 I've hardcoded in the `terragrunt.hcl` to be EU as I there are a ton of other places where location is a parameter that defaults to EU in the code.